worldedit_commands/init.lua

```diff
@@ -465,6 +465,30 @@ local check_sphere = function(name, param)
 		worldedit.player_notify(name, "invalid usage: " .. param)
 		return nil
 	end
+	if nil ~= area_protection.areas then
+		local pos1 = worldedit.pos1[name]
+		local allowed, conflicting = area_protection.areas:canInteractInArea(
+			{
+				x = pos1.x - radius,
+				y = pos1.y - radius,
+				z = pos1.z - radius,
+			},
+			{
+				x = pos1.x + radius,
+				y = pos1.y + radius,
+				z = pos1.z + radius,
+			},
+			name,
+			false
+		)
+		if false == allowed then

```

Comment:
Considering that the equality check operator has commutative semantics while human language semantics often don't, I don't find the "Yoda" comparison and its connotations appropriate. I'm open to objective critique on the code quality, though...